docs/_static/custom.css

```diff
@@ -1 +1,129 @@
-#version-picker-label { color: white; display: inline; }
+@import url("https://fonts.googleapis.com/css?family=Space+Mono&display=swap");
```

Comment:
This is work from @ahoymrag 
There is another PR open for this already but it is getting stale, I mentioned to him that I am revamping the docs and he sent me the CSS so I am just throwing it in here to get this through because it is amazing.

I did run `prettier --write` on it though.